lib/checks/canMultiplyLikeTermConstantNodes.js

```diff
@@ -0,0 +1,30 @@
+const Node = require('../node');
+
+
```

Comment:
just to match what other files are doing right now, can you do only one new line between the require and the comment, and no new lines at all between the comment and function? ^_^